azure-pipelines.yml

```diff
@@ -248,6 +248,12 @@ stages:
             FrameworkMoniker: 'net8.0'
 
       steps:
+      - task: UseDotNet@2
+        displayName: Use .NET
+        inputs:
+          packageType: 'sdk'
+          version: 8.0.300
```

Comment:
Educational: what is this about?

Reply:
You answered [here](https://github.com/SonarSource/sonar-dotnet/pull/9295#discussion_r1604937061). All good on my side